roles/haproxy/templates/haproxy_frontend.cfg.j2

```diff
@@ -83,6 +86,9 @@ frontend internet_restricted_ip
     capture request header User-agent len 256 
     # Put the useragent header in a variable, shared between request and response. 
     http-request set-var(txn.useragent) req.fhdr(User-Agent)
+    # The ACL below makes sure only supported http methods are allowed
+    acl valid_method method {{ haproxy_supported_http_methods }}
+    http-request deny if !valid_method
```

Comment:
Returning a 405 seems to match the situation better than 403.

Reply:
You're right. Fixed it!